labs/nft-community-footprint.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: NFT Community Footprint
+parent: Labs
+---
+# Lab Name
+nft-community-footprint
+
+# Short Description
+Mint tokens for contributors' foot prints in fabric community.
+
+# Scope of Lab
+Open source projects with warm, welcoming communities keep people coming back for years. Many people form lifelong friendships through their participation in open source, whether at conferences or late night online chats. Contributors are like the blood of open source projects because they enable smaller projects to grow without a lot of financial support and they bring new ideas to their projects.
+
+We want to mint tokens for contributors' foot prints in fabric community. This can be an amazing experience. By this way, their contributing to open source can be like a reward and that becomes a part of communities in NFT.
```

Comment:
```suggestion
We want to mint tokens for contributors' foot prints in the Fabric community. This can be an amazing experience. By this way, their contributions to open source can be like a reward and that becomes a part of communities in NFT.
```